apps/src/music/utils/Tunes.ts

```diff
@@ -69,3 +77,74 @@ export function generateGraphDataFromTune({
     };
   });
 }
+
+export function getDisplayNotes(
+  editorType: EditorType,
+  scaleMode: ScaleMode,
+  instrument: string,
+  rootKey: Key
+) {
+  if (editorType === 'drums') {
+    const kitFolder = MusicLibrary.getInstance()?.kits.find(
+      kit => kit.id === instrument
+    );
+    return (
+      kitFolder?.sounds.map((sound, i) => ({
+        name: sound.name,
+        note: i,
+        colors: null,
```

Comment:
Is this used somewhere? It looks like this is the only change to the `getDisplayNotes` function (aside from moving it), but I'm not sure what it's for.

Reply:
Good catch.  No longer needed.